packages/ui/vite.config.ts

```diff
@@ -30,7 +30,16 @@ export default defineConfig({
       formats: ['es']
     },
     rollupOptions: {
-      external: external
+      external: external,
+      output: {
+        assetFileNames: chunkInfo => {
+          if (chunkInfo.name === 'style.css') {
+            return 'styles.css'
```

Comment:
why do we need to rename the output? cant we just use `style.css` when importing everywhere?